spec/requests/api/v1/users_spec.rb

```diff
@@ -19,6 +19,21 @@
         }
       }
 
+      it "create a client_session record" do
+        expect{
+          post path, params: params
+        }.to change(ClientSession, :count).by(1)
+      end
+
+      it "returns session data with access token" do
+        post path, params: params
+        expect(response).to have_http_status(:created)
+        expect(json["data"]["relationships"]["sessions"]["data"][0]["id"]).to be
+        expect(json["data"]["relationships"]["sessions"]["data"][0]["type"]).to eq "session"
+        expect(json["meta"]["access_token"]).to be
```

Comment:
missing what it should equal -- why is this passing?

Reply:
@littleforest it should just the opposite of `.to_not be_nil`. I wasnt testing for a certain value, just that it is returning one.